verilog/tools/ls/verilog-language-server.cc

```diff
@@ -33,14 +34,45 @@ VerilogLanguageServer::VerilogLanguageServer(const WriteFun &write_fun)
   // Whenever there is a new parse result ready, use that as an opportunity
   // to send diagnostics to the client.
   buffers_.SetChangeListener(parsed_buffers_.GetSubscriptionCallback());
-  parsed_buffers_.SetChangeListener(
+  parsed_buffers_.AddChangeListener(
       [this](const std::string &uri,
              const verilog::BufferTracker &buffer_tracker) {
         SendDiagnostics(uri, buffer_tracker);
       });
   SetRequestHandlers();
 }
 
+verible::lsp::InitializeResult VerilogLanguageServer::GetCapabilities() {
+  // send response with information what we do.
+  verible::lsp::InitializeResult result;
+  result.serverInfo = {
+      .name = "Verible Verilog language server.",
+      .version = GetVersionNumber(),
```

Comment:
FYI, upstream in #1624  this one call changed from `GetVersionNumber()` changed to `verible::GetRepositoryVersion()`.

Because the block moved in the file, git can't resolve this and end up in merge conflict that looks more nasty than it actually is. So you'll have to manually change this function call.